cachito/web/static/api_v1.yaml

```diff
@@ -228,8 +232,6 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/RequestNew'
-      security:
```

Comment:
Why was this removed?

Reply:
Forgot to mention this: we get this error when security is present:

    ... Unsupported http authorization scheme negotiate

Talking to @taylormadore yesterday, we started thinking that maybe connexion probably doesn't need to know about or care about kerberos (kerberos isn't even supported by OpenAPI). What do you think @MartinBasti?

Reply:
I got this test running locally changing this:
```
diff --git a/cachito/web/api_v1.py b/cachito/web/api_v1.py
index d840d86..0c51b55 100644
--- a/cachito/web/api_v1.py
+++ b/cachito/web/api_v1.py
@@ -153,16 +153,16 @@ def get_requests():
     return flask.jsonify(response)
 
 
-def get_request(request_id):
+def get_request(id):
     """
     Retrieve details for the given request.
 
-    :param int request_id: the value of the request ID
+    :param int id: the value of the request ID
     :return: a Flask JSON response
     :rtype: flask.Response
     :raise NotFound: if the request is not found
     """
-    json = Request.query.get_or_404(request_id).to_json()
+    json = Request.query.get_or_404(id).to_json()
 
     if json["state"] == RequestStateMapping.complete.name:
         package_count = len(json["packages"])
@@ -171,7 +171,7 @@ def get_request(request_id):
         flask.current_app.logger.info(
             "Returning data for request %i. Found %i packages and %i dependencies. "
             "The following package managers were used: %s.",
-            request_id,
+            id,
             package_count,
             dependency_count,
             json["pkg_managers"],
diff --git a/cachito/web/models.py b/cachito/web/models.py
index 8c0523a..7058608 100644
--- a/cachito/web/models.py
+++ b/cachito/web/models.py
@@ -439,13 +439,13 @@ class Request(db.Model):
 
         if verbose:
             rv["configuration_files"] = flask.url_for(
-                "api_v1.get_request_config_files", request_id=self.id, _external=True
+                "/api/v1.cachito_web_api_v1_get_request_config_files", id=self.id, _external=True
             )
             rv["content_manifest"] = flask.url_for(
-                "api_v1.get_request_content_manifest", request_id=self.id, _external=True
+                "/api/v1.cachito_web_api_v1_get_request_content_manifest", id=self.id, _external=True
             )
             rv["environment_variables_info"] = flask.url_for(
-                "api_v1.get_request_environment_variables", request_id=self.id, _external=True
+                "/api/v1.cachito_web_api_v1_get_request_environment_variables", id=self.id, _external=True
             )
             # Use this list comprehension instead of a RequestState.to_json method to avoid
             # including redundant information about the request itself
@@ -469,7 +469,7 @@ class Request(db.Model):
             if flask.current_app.config["CACHITO_REQUEST_FILE_LOGS_DIR"]:
                 rv["logs"] = {
                     "url": flask.url_for(
-                        "api_v1.get_request_logs", request_id=self.id, _external=True
+                        "/api/v1.cachito_web_api_v1_get_request_logs", id=self.id, _external=True
                     )
                 }
         else:
```

Reply:
I think that such change should be mentioned in commit message with explanation why

Reply:
@brunoapimentel can you pls explain me, how your diff is related to krb negotiate auth? I miss this magic

Reply:
I posted it to the wrong comment :sweat_smile:. (Deleted it to avoid further confusion)